tests/start.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd autohat
+MACHINE=${JOB_NAME#yocto-}
```

Comment:
I would use here device type to not confuse it with yocto machine. Even though they should be the same, there are exceptions (ex rpi).